### DISCUSSION TITLE
Renamed Folder - All Pages GONE

### DISCUSSION THREAD

**Discussion by Wikojs** [requarks/wiki#5509] [Error / Bug Report]
I renamed Folder "Development" into "Entwicklung" (see Attachement )

![Folder rename 1 ](https://user-images.githubusercontent.com/102657017/181283539-500001b8-9992-4d83-aab4-7f36917a8e11.png)

![Folder rename](https://user-images.githubusercontent.com/102657017/181282411-46d8bfc5-5f3c-4177-8384-2b5dc98979f0.png)

Now are all pages under this Folder gone!
The thing is the docker needed;
systemctl restart docker.socket
because he was on and there, but communication was broken, and wiki Docker dumped a crontab backup with 0 kb, therefore I dont have a backup.

I tried to rename the folder back to "Development" to see if the pages can find their way back, but they didnt pop up again... 

please Gods of IT is there any trick here that can help me?

**Comment by Superfrank1186** [comment]
Sorry for the  interruption but I  need immediate help with this neighbors of mine that have been cyberstalking me for over 4 years  they harassed and threatened me daily that they are going  to rape my wife and daughter I can't even  go to work comfortable thinking this criminals could hurt my family they have some kind of surveillance that follows me around everywhere I go and still can't figure out what it is  and I'm sick and tired  of this I already called  the police and did nothing because needed proof and I also made 3 ic3 reports threw the FBI and still have  not gotten  a response  so I please ask for your immediate help to put this criminals behind bars thank you for taking this into consideration and god bless.
